gdk/common/utils.py

```diff
@@ -226,6 +227,11 @@ def parse_json_schema_errors(error):
             logging.info(f"\t {fix}")
 
 
+def valid_recipe_file_size(file_path):
+    file_size = os.path.getsize(file_path)
+    return file_size < 16234    # max allowed byte
```

Comment:
nit: Rather than noting the purpose of the literal in a comment, I think it's better to write it as a constant with an appropriate name. Because a comment is more likely to be ignored and not updated if the code is changed later, and this way the code reads more naturally in my opinion.

 e.g. 

```
MAX_RECIPE_FILE_SIZE_BYTES = 16234
file_size_bytes = os.path.getsize(file_path)
return file_size_bytes < MAX_RECIPE_FILE_SIZE_BYTES
```